lib/produce/config.rb

```diff
@@ -12,25 +12,28 @@ def initialize
         :app_name => ENV['PRODUCE_APP_NAME'],
         :primary_language => ENV['PRODUCE_LANGUAGE'],
         :version => ENV['PRODUCE_VERSION'],
-        :sku => ENV['PRODUCE_SKU']
+        :sku => ENV['PRODUCE_SKU'],
+        :skip_itc => ENV['PRODUCE_SKIP_ITC']
```

Comment:
Is there any reason you don't use new Hash syntax here?

Btw, let me know if you'd like, and I'll happily send PR for that!